### ISSUE TITLE
Support batch operations

### ISSUE THREAD

**Issue by cdauth** [FacilMap/facilmap#160]
Support batch updates of data fields for markers and lines.

**Comment by lost-geographer** [comment]
Do you mean, for example, adding the possibility to change the type of a group of selected markers ? That would be so cool !

As far as I know, for now we can only remove a group of selected markers. Am I right ?

![Capture d’écran du 2022-03-08 14-15-01](https://user-images.githubusercontent.com/11869843/157246001-f78c372b-3d4a-4a2e-9b09-9d0de6047a13.png)

**Comment by cdauth** [comment]
Yes, that's exactly the plan. To be able to set the type or a field value or a style attribute of a group of markers/lines.

**Comment by georg-d** [comment]
FYI a workaround: I exported, edited the text file as required with search & replace, then re-imported.
